server/models/handlers/account.js

```diff
@@ -1,22 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
-        type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
-        studentId: '20764242',
-        firstname: 'Victor',
-        lastname: 'Hugo',
-        email: 'victorhugo@gmail.com',
-        phone: '6476442200'
+async function createUserAccount(firstName, lastName, type, studentId, email, password, phone, userType, workerId, specialization, schoolId ) {
+    // Searches the student table to see if a student account exists for the given data.
+    var user;
+    if (userType == userType.student) {
+        //Generates an accessToken of length 20 characters.
+        // Note: The accessToken includes all alphanumeric characters except for 0, O, I, and l — characters easily mistaken for each other)
+        const accessToken = new UIDGenerator(UIDGenerator.BASE16,20);
```

Comment:
Hi Nancy,
I would recommend moving this to line 7 (outside of the if statement so the else block can also access the `accessToken` variable as well.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've moved the `UIDGenerator` function outside the if statement.

You can now see these changes in line 7 of server/models/handlers/account.js

Please confirm if this takes care of your suggestion!